### DISCUSSION TITLE
Table with subcomponents

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents#6841] [Q&A]
Is it currently possible to have row subcomponents in the Table component, like the AnalyticalTable does?  If not, do you have any suggestions for implementing this with custom code?

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/127273220/228725964-18b07758-eccb-46c5-86f0-0387c9fab4ab.png">

**Comment by MarcusNotheis** [comment]
Hi @qscheitlin,
the `<Table />` component is developed in the UI5 Web Components repository, so I'll transfer this discussion to the corresponding repo.

**Comment by petyabegovska** [comment]
Hi @qscheitlin,

This is not possible with the Table component. Can you give us more details on the use case for adding such a subcomponent?

Best regards,
Petya
